resources/lib/pigskin.py

```diff
@@ -334,7 +335,7 @@ def get_publishpoint_streams(self, video_id, stream_type=None, game_type=None, u
         request = self.make_request(videoDataPath, 'get')
         akamai_url = xmltodict.parse(request)
         for videoSource in akamai_url['video']['videoSources']['videoSource']:
-            if videoSource['@format']== 'HLS':
+            if videoSource['@format']== 'ChromeCast':
```

Comment:
When working on multiple fixes, 'git add -p' is a wonderful for choosing exactly which bits you want for a given commit.

---Alex